### DISCUSSION TITLE
Missing my plots

### DISCUSSION THREAD

**Discussion by kumaarsam** [Chia-Network/chia-blockchain#6279] [Support]
Hello Chia Team,

I am writing this to let you know that I am missing my plot on network space. I had placed over 13 plots and Chia never recognised 5 of them but yesterday it cut down another 3. I can see them in my drives but they are not on Chia application.

Also it is taking too much CPU and RAM while plotting, that sometimes even make my desktop freeze. 
![2021-05-27 (8)](https://user-images.githubusercontent.com/85018755/120243036-66107100-c25e-11eb-8419-50486982ca88.png)
![2021-05-28 (1)](https://user-images.githubusercontent.com/85018755/120243067-83ddd600-c25e-11eb-9fd2-bb5751cee503.png)
![2021-05-28 (2)](https://user-images.githubusercontent.com/85018755/120243096-99eb9680-c25e-11eb-9117-2a4cfda78b5d.png)
![2021-05-28 (3)](https://user-images.githubusercontent.com/85018755/120243159-c9020800-c25e-11eb-9c8c-55bcb137e852.png)
![2021-05-28 (7)](https://user-images.githubusercontent.com/85018755/120243190-dc14d800-c25e-11eb-9d40-750e72a89ffd.png)
![2021-05-28 (9)](https://user-images.githubusercontent.com/85018755/120243220-f058d500-c25e-11eb-9c18-990d625a7121.png)
![2021-05-28 (10)](https://user-images.githubusercontent.com/85018755/120243262-0c5c7680-c25f-11eb-9b39-48d3636325f1.png)
![2021-05-28 (11)](https://user-images.githubusercontent.com/85018755/120243264-0cf50d00-c25f-11eb-96a8-e317d5c97a2e.png)
![2021-05-28 (12)](https://user-images.githubusercontent.com/85018755/120243296-239b6400-c25f-11eb-9a8e-7dc208720e0d.png)
![2021-05-31 (2)](https://user-images.githubusercontent.com/85018755/120243391-5b0a1080-c25f-11eb-94c9-21ce4226317e.png)

Could you please help with it?

**Comment by BrandtH22** [comment]
@kumaarsam , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Note - the plotting process is resource intensive and it is recommended to only have the plotter running (close onedrive and chrome). The desktop freezing while plotting can result in bad or invalid plots. Lastly, you may want to remove and readd the plot directories to force the client to rescan them.

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
